de_web_plugin.cpp

```diff
@@ -3059,7 +3043,9 @@ void DeRestPluginPrivate::addLightNode(const deCONZ::Node *node)
 
         if (existDevicesWithVendorCodeForMacPrefix(node->address(), VENDOR_PHILIPS))
         {
-            if (lightNode.manufacturer() != QLatin1String("Philips"))
+            if (lightNode.manufacturer() != QLatin1String("Philips") &&
+                lightNode.manufacturer() != QLatin1String("Signify Netherlands B.V.")
+               )
             { // correct vendor name, was atmel, de sometimes
```

Comment:
Note for the future we need to get rid of this. The Device code already robustly queries the Basic cluster manufacturer name, no mac prefix magic needed.

Reply:
So leave it for now?

Reply:
Thinking about it a bit more, I'd suggest we can remove this code. The DDFs should use the actual exposed manufacturer name from the *Basic Cluster*, it can be cumbersome to have "Philips" entries in the database which should be "Signify Netherlands B.V.".